src/workload.py

```diff
@@ -155,15 +148,37 @@ def get_service_pid(self) -> int:
         for pid in java_processes.split():
             with open(f"/proc/{pid}/cgroup", "r") as fid:
                 content = "".join(fid.readlines())
+                logger.error(content)
+                logger.error(f"{self.SNAP_NAME}.{self.service}")
```

Comment:
**todo:** If we still want to keep this (i.e it wasn't just debugging), let's wrap these together in to a single message.

Reply:
Done in 3809a7ee84bf52180b1ba6c0559c265bc2559d90